src/client/Microsoft.Identity.Client/ApiConfig/AcquireTokenInteractiveParameterBuilder.cs

```diff
@@ -302,17 +303,14 @@ public AcquireTokenInteractiveParameterBuilder WithParentActivityOrWindow(IWin32
         /// <summary>
         /// Sets a reference to the IntPtr to a window that triggers the browser to be shown.
         /// Used to center the browser that pop-up onto this window.
+        /// The center of the screen or the foreground app if a value is configured.
         /// </summary>
-        /// <param name="window">The current window</param>
+        /// <param name="window">The current window as IntPtr</param>
         /// <returns>The builder to chain the .With methods</returns>
+        /// <remarks></remarks>
         [CLSCompliant(false)]
         public AcquireTokenInteractiveParameterBuilder WithParentActivityOrWindow(IntPtr window)
         {
-            if (window == null)
-            {
-                throw new ArgumentNullException(nameof(window));
-            }
-
             return WithParentObject((object)window);
```

Comment:
why is this one okay to be null but the one w/`IWin32Window` is not? if i can be null, then you could set the parameter to null and it wouldn't be required to be pass in. like on iOS, how we handle it internally.

Reply:
IntPtr is a struct, just like int, and a struct cannot be null, i.e. that `if` clause was pointless. The default value of IntPtr is IntPtr.Zero

Reply:
makes sense. ty,

---
In reply to: [537353165](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/2279#discussion_r537353165) [](ancestors = 537353165)